aif360/metrics/utils.py

```diff
@@ -126,6 +120,29 @@ def compute_num_TF_PN(X, y_true, y_pred, w, feature_names, favorable_label,
         FN=np.sum(w[np.logical_and(y_true_pos, y_pred_neg)], dtype=np.float64)
     )
 
+def compute_ROC(y_true, scores, pos_label = 1, condition = None):
```

Comment:
you should use `favorable_label` here like above instead of `pos_label`.

also, `condition` should be implemented like above too (it's not currently used here)